LeftNavBarExample/LeftNavBarLibrary/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.0.0'
+        classpath 'com.android.tools.build:gradle:2.2.1'
```

Comment:
Nah, duplication of the wrapper can't be avoided, but I really don't feel is like a big deal 😸  Actually this is a good opportunity to use Gradle 3.x. in your own _isolated and cohesive_ project. Mmm 🤔  sounds even better to me.